### ISSUE TITLE
[E2E] identify more e2e tests specific for Osmosis and write up issues to add them

### ISSUE THREAD

**Issue by p0mvn** [osmosis-labs/osmosis#1202]
**Background**

We should start adding new tests once #1201 is complete with existing tests ported from other repositories. However, we can begin identifying new test cases that we would like to add in the future.

**Acceptance Criteria**
- [ ] e2e test plan is created
- [ ] new issues are written up for the next sprint.

**Comment by p0mvn** [comment]
I'm not sure how this could look like yet but it would be great to performance test and send thousands of queries to an e2e relayer to try and catch app hash in these tests

**Comment by p0mvn** [comment]
Based on the problems experienced in: https://github.com/osmosis-labs/osmosis/issues/1356

We should have nodes with:
- grpc enabled
  * [ ] query old height
  * [ ] query latest height
  * [ ] do not provide a height 
- grpc disabled
  * [ ] query old height
  * [ ] query latest height
  * [ ] do not provide a height

**Comment by mattverse** [comment]
Hey @p0mvn  do you think this issue can be closed since we have https://github.com/osmosis-labs/osmosis/issues/1235 or would you define it a separate issue?

**Comment by czarcas7ic** [comment]
I would say this issue can be closed, because at the very least we have https://github.com/osmosis-labs/osmosis/issues/1605
